src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -144,52 +215,79 @@ sealed trait LazyModuleImpLike extends RawModule
   val auto: AutoBundle
   protected[diplomacy] val dangles: Seq[Dangle]
 
-  // .module had better not be accessed while LazyModules are still being built!
-  require (!LazyModule.scope.isDefined, s"${wrapper.name}.module was constructed before LazyModule() was run on ${LazyModule.scope.get.name}")
+  /** [[wrapper.module]] had better not be accessed while LazyModules are still being built! */
+  require (LazyModule.scope.isEmpty, s"${wrapper.name}.module was constructed before LazyModule() was run on ${LazyModule.scope.get.name}")
 
+  /** set name from [[wrapper]]. */
   override def desiredName = wrapper.desiredName
   suggestName(wrapper.suggestedName)
 
-  implicit val p = wrapper.p
+  /** [[Parameters]] for diplomatic [[Module]]s. */
```

Comment:
is diplomatic [[Module]]s just [[LazyModule]]? If so, should we just use that instead?

Reply:
changed to
```
  /** [[Parameters]] for chisel [[Module]]s. */
```